packages/SwingSet/src/liveslots/virtualObjectManager.js

```diff
@@ -572,7 +572,18 @@ export function makeVirtualObjectManager(
     options,
     durable,
   ) {
-    const finish = options ? options.finish : undefined;
+    let finish;
+    let fakeDurable;
+    if (options) {
+      ({ finish, fakeDurable } = options);
+    }
+    if (fakeDurable) {
+      assert(
+        durable,
```

Comment:
A bit surprising since collectionManager uses flags of exactly these two names but insists that those are mutually exclusive. I don't have a suggestion, so it you don't think of an improvement, you can leave this as is.